class ValueException(Exception):
    pass

class RepoException(Exception):
    pass